### PR TITLE
config_tools: add interface to save config xmls to user defined path …

### PR DESCRIPTION
**Pull request by szhen11**
…in config editor

add interface to save scenario xmls and launch xmls to user defined
path in config editor; move all config xmls and generated scripts
out of acrn-hypervisor.

Tracked-On: #6208
Signed-off-by: Shuang Zheng <shuang.zheng@intel.com>
Reviewed-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Review by NanlinXie — APPROVED** [review]

File: misc/config_tools/config_app/static/main.js [hunk 1]
```diff
@@ -790,7 +789,7 @@ function create_setting(type, default_name, name, mode){
             create_flag = true
             if(exist == "yes") {
                 overwirte_confirm_message = 'Setting name: ' + create_config['create_name'] + ' existed in ' +
```
Comment (dbkinder) [inline misc/config_tools/config_app/static/main.js:791]: the word overwrite_ is misspelled as overwirte_ in the code here

**Review by dbkinder — COMMENTED** [review]
I suspect this change will have an impact on documentation...

File: misc/config_tools/config_app/static/main.js (hunk 1 above)
Comment (szhen11) [reply misc/config_tools/config_app/static/main.js]: Thanks for the reminder. updated this commit and fixed the typo.

**Comment by NanlinXie** [comment]
@dbkinder , we will check and update configtool document in the coming v2.6 release.

**Pull request merged**